src/components/activityEventElems/ActivityElement/ActivityElement.tsx

```diff
@@ -34,43 +61,53 @@ const ExtraContainer: React.FC = ({ children }) => {
   )
 }
 
-function Header({ header }: { header: string }) {
+function Header({ header }: { header: string | JSX.Element | null }) {
   const {
     theme: { colors },
   } = useContext(ThemeContext)
-  return (
-    <div style={smallHeaderStyle(colors)}>
-      <Trans>{header}</Trans>
-    </div>
-  )
+  return <div style={smallHeaderStyle(colors)}>{header}</div>
 }
 
-function SideDetails({ event }: { event: ActivityElementEvent }) {
+function SideDetails({ details }: { details: Partial<ActivityElementEvent> }) {
```

Comment:
little nit: can we use `Pick<>` to specify which props are needed, instead of the vague `Partial<>`